### PR TITLE
 Add to Cart functionality for products

### PR DESCRIPTION
**Pull request by YoussefHamadaYH**

### CODE REVIEW

**Comment by YoussefHamadaYH** [comment]
feat: Created "Add to Cart" functionality for products, allowing users to add items to their shopping cart

File: Book-Store-MVC/Controllers/CartController1.cs [hunk 1]
```diff
@@ -0,0 +1,60 @@
+﻿using AutoMapper;
+using Book_Store_MVC.Models;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Book_Store_MVC.Controllers
+{
+    public class CartController1 : Controller
+    {
+        private readonly BookStoreContext bookStore;
```
Comment (Diab16) [inline Book-Store-MVC/Controllers/CartController1.cs]: use the repo not the DBset

**Review by Diab16 — COMMENTED** [review]
stop using context

**Comment by YoussefHamadaYH** [comment]
Using Repositaroy instead of Dbset

**Pull request merged**